CONTRIBUTING.md

```diff
@@ -0,0 +1,284 @@
+He makes the final decision on both design and code changes within Bevy in order to ensure a coherent vision and consistent quality of code.
+
+In practice, @cart serves as a shockingly accountable dictator: open to new ideas and to changing his mind in the face of compelling arguments or community consensus.
+Check out the next section for details on how this plays out.
+
+[Bevy Org members](https://github.com/orgs/bevyengine/people) are contributors who have:
+
+1. Have actively engaged with Bevy development
+2. Have demonstrated themselves to be polite and welcoming representatives of the project with an understanding of our goals and direction.
+3. Have asked to join the Bevy Org. Reach out to @cart on Discord or [email us](bevyengine@gmail.com) if you are interested. Everyone is welcome to do this. We generally accept membership requests, so don't hesitate if you are interested!
+
+Some Bevy Org members are also [Triage Team](https://github.com/orgs/bevyengine/teams/triage-team) members. These people can label and close issues and PRs but do not have merge rights or any special authority within the community. Existing Bevy Engine Org members can [automatically request membership](https://github.com/orgs/bevyengine/teams/triage-team/members). Once again, if you are interested don't hesitate to apply. We generally accept membership requests.
+
+We heavily limit who has merge rights within the org because this requires a large amount of trust when it comes to ethics, technical ability, and ability to enforce consistent project direction. Currently, only @cart can merge every class of change. @mockersf is allowed to merge small "uncontroversial" changes, provided they have two approvals.
+
+## How we work together
+
+Making a game engine is a huge project and facilitating collaboration is a lot of work. At the moment @cart is our only paid contributor, so [go sponsor him!](https://github.com/sponsors/cart).
+
+While we have *many* active contributors, herding all of these ~~cats~~ birds in a predictable way is challenging.
+
+Bevy releases are intended to be spaced 6-8 weeks apart and tend to target one or two major features, led by @cart.
+[Once those features are complete](https://github.com/bevyengine/bevy/blob/main/docs/release_checklist.md), we work to fix any [P-High](https://github.com/bevyengine/bevy/issues?q=is%3Aopen+is%3Aissue+label%3AP-High), [P-Critical](https://github.com/bevyengine/bevy/issues?q=is%3Aopen+is%3Aissue+label%3AP-Critical) or [C-Regression](https://github.com/bevyengine/bevy/issues?q=is%3Aopen+is%3Aissue+label%3AC-Regression) labeled issues that we can, write up our release notes and migration guide, and then announce the next Bevy version to the world!
+
+We track issues and pull requests that must be included in releases using [Milestones](https://github.com/bevyengine/bevy/milestones).
+
+You can see what we're planning by following along at the [Bevy Roadmap](https://github.com/bevyengine/bevy/projects/1).
```

Comment:
The project board (and the pinned issues) are a little out of date I think. Linking to this here might give new users the wrong idea about how active Bevy development is. I remember another open source project which had a blog that was never updated. It made the project look dead to new users, even though it was very much alive.

Reply:
Alice and I plan on giving the project board(s) some love prior to the next release. I think I'm fine with linking to it for now, but I agree with your point.